osu.Game/OsuGame.cs

```diff
@@ -417,7 +416,16 @@ protected override void LoadComplete()
                                     screenStack.Exit();
                             }
                         },
-                        logoContainer = new Container { RelativeSizeAxes = Axes.Both },
+                        new Container
```

Comment:
Is there a particular reason to add this container level?

Reply:
This is resolving the original issue by moving the back button one level higher than the screen stack, while allowing the back button to remain contained inside the `screenContainer`. Not sure if there's another way to achieve this.

Reply:
It doesn't work with just ordering?

Reply:
didn't think of that, seems to work fine.